src/routes/bugListRoute.ts

```diff
@@ -1,8 +1,10 @@
 import express from 'express';
-import { getBugPosts, getBugPostDetails } from '@/controllers/bugListController';
+import { getBugReportList, getBugReportDetail } from '@/controllers/bugListController';
+import { authenticateToken } from '@/middlewares/authMiddleware';
+
 const router = express.Router();
 
-router.get('/catch-requests', getBugPosts);  // 사냥 리스트 조회
-router.get('/catch-requests/more', getBugPostDetails);  // 사냥 상세 정보 조회
+router.get('/bug/list', getBugReportList);  // 사냥 리스트 조회
```

Comment:
p1; 제가 엔드포인트를 헷갈려서 잘못 적었네요.. 아래의 내용으로 수정 부탁드리겠습니다.
```suggestion
router.get('/bug-reports', getBugReportList);  // 사냥 리스트 조회
```